src/routes/sections.jsx

```diff
@@ -47,6 +49,10 @@ export default function Router() {
       path: 'forgot-password',
       element: <ForgotPasswordPage />,
     },
+    {
+      path: 'oauth-callback',
+      element: <OAuthCallback />,
+    },
```

Comment:
Add React Suspense with Skeleton from tailwind since you already  used lazy loading.

Reply:
Okay